### ISSUE TITLE
TigerVNC not showing background and menu bars

### ISSUE THREAD

**Issue by tguk70** [TigerVNC/tigervnc#556]
Hi

I am running tigervnc-server on a cent os 7 running the latest updates. I have installed vncserver so it starts on a standard user, but when I connect, there are no menu bars or the standard background. I have installed tigervnc-server-1.8.0-2.el7_4.x86_64

The only thing in the log for the connection is: 
SConnection: Client need protocol version 3.8 
SConnection: Client requests security type VncAuth(2)

**Comment by CendioOssman** [comment]
That looks like Red Hat's packaging, which unfortunately differs from ours in this regard. So it might be a problem with that.

Exactly how did you configure the startup?

And what do you see? Just a blank background?

**Comment by tguk70** [comment]
Thank you for your reply.
I copied as root, the vncserver@.service from /lib/systemd/system/vncserver@.service to /etc/systemd/system/vncserver@.service

Using vi, i edited the file in /etc/systemd/system/vncserver@.service replacing <USER> with my relevant username. I then run su - <USER> and created a vnc password for them.
Then exited the user environment back into root. Ran systemctl daemon-reload, then systemctl start vncserver@:1.service and systemctl enable vncserver@:1.service.

When i connect, the password is accepted and i can open a terminal window by right clicking, but i cannot switch between windows or browser.

Hope this gives you more insight.

Many Thanks

![screen shot 2017-12-11 at 18 13 02](https://user-images.githubusercontent.com/6271984/33846479-f889374e-de9e-11e7-8a81-2fd400aadbef.png)
